package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kava-labs/javascript-sdk",
-  "version": "10.0.0-beta-1",
+  "version": "14.0.0-beta",
```

Comment:
Skipping straight to 14 here so it coincides with the kava mainnet releases. Should be easier on 3rd parties to match the versioning of the sdk with the correct stuff on chain.